.npmrc

```diff
@@ -0,0 +1,7 @@
+# rollup-plugin-dts needs packages to be hoisted
```

Comment:
Is it really mandatory ?

![image](https://user-images.githubusercontent.com/62407/158965798-b5a181e9-c82d-452b-9740-5d3b26fd77de.png)

I got no error, and types seems fine
but I'm not sure about 
> rollup-plugin-dts needs packages to be hoisted

does it trigger error or introduce broken definition in types?

Reply:
Ok, I got an error, this is required to build ;)
![image](https://user-images.githubusercontent.com/62407/158966845-2f9be7cf-cb07-4758-8ea9-f57267129026.png)
once I trashed `node_modules` and made a fresh reinstall with `pnpm`!

=> It may **deserve more visibility** (I won't look there at first, when having a build error),
**maybe a "troubleshouting" section in readme**.